llvm/lib/IR/ProfileSummary.cpp

```diff
@@ -251,17 +251,18 @@ ProfileSummary *ProfileSummary::getFromMD(Metadata *MD) {
 void ProfileSummary::printSummary(raw_ostream &OS) const {
   OS << "Total functions: " << NumFunctions << "\n";
   OS << "Maximum function count: " << MaxFunctionCount << "\n";
-  OS << "Maximum block count: " << MaxCount << "\n";
+  OS << "Maximum internal block count: " << MaxInternalCount << "\n";
```

Comment:
I also changed the behavior here to dump `MaxInternalCount` instead of `MaxCount` because I think it requires fewer test changes. But I'm also open to changing this or including both logs.

Reply:
What is "internal block" and what differentiate it from just "block"?

Reply:
I believe an "internal block" is a non-entry blocks while "block" includes the entry block, which is is reported in "function count".